src/dhcpmon/src/dhcp_device.cpp

```diff
@@ -252,62 +308,28 @@ static void read_callback_dual_tor(int fd, short event, void *arg)
     while ((event == EV_READ) &&
```

Comment:
Here "event == EV_READ" will always true as we register this handler for EV_READ, we don't need to check it again.